ml.net/InclusiveCodeReviews.Scoring/ModelBuilder.cs

```diff
@@ -0,0 +1,195 @@
+//*****************************************************************************************
+//*                                                                                       *
+//* This is an auto-generated file by Microsoft ML.NET CLI (Command-Line Interface) tool. *
+//*                                                                                       *
+//*****************************************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using InclusiveCodeReviews.Model;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms.Text;
+
+namespace InclusiveCodeReviews.ConsoleApp
+{
+	public static class ModelBuilder
+	{
+		private static string TRAIN_DATA_FILEPATH = Path.Combine(Path.GetDirectoryName(typeof(ModelBuilder).Assembly.Location), "..", "..", "..", "..", "..", "comments", "classified.csv");
+		private static string MODEL_FILE = ConsumeModel.MLNetModelPath;
```

Comment:
Can you add a project reference instead of copying this file?

Or a "link" to the source file would also work.

Reply:
removed ModelBuilder referring from ConsoleApp